source/package-lock.json

```diff
@@ -1,8 +1,1178 @@
 {
```

Comment:
Probably want to remove this from the pull request, as it might cause merge issues.